test/units/modules/network/netvisor/test_pn_vrouter_pim_config.py

```diff
@@ -0,0 +1,56 @@
+# Copyright: (c) 2018, Pluribus Networks
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+from units.compat.mock import patch
+from ansible.modules.network.netvisor import pn_vrouter_pim_config
+from units.modules.utils import set_module_args
+from .nvos_module import TestNvosModule, load_fixture
+
+
+class TestVrouterPimConfigModule(TestNvosModule):
+
+    module = pn_vrouter_pim_config
+
+    def setUp(self):
+        self.mock_run_nvos_commands = patch('ansible.modules.network.netvisor.pn_vrouter_pim_config.run_cli')
+        self.run_nvos_commands = self.mock_run_nvos_commands.start()
+
+        self.mock_run_check_cli = patch('ansible.modules.network.netvisor.pn_vrouter_pim_config.check_cli')
+        self.run_check_cli = self.mock_run_check_cli.start()
+
+    def tearDown(self):
+        self.mock_run_nvos_commands.stop()
```

Comment:
```suggestion
        self.mock_run_nvos_commands.stop()
        self.run_check_cli.stop()
```

Reply:
Sorry @rajaspachipulusu17 I think this should be `self.mock_run_check_cli.stop()`

Reply:
@Akasurde its already `self.mock_run_check_cli.stop()` what should i change? i didnt get it